cirras-underwriting-api/cirras-underwriting-service-api/src/main/java/ca/bc/gov/mal/cirras/underwriting/service/api/v1/impl/CirrasInventoryServiceImpl.java

```diff
@@ -500,18 +500,25 @@ private Double getUnseededAcres(InventoryContractCommodity inventoryContractComm
 			// Get sum of acres of commodities that have a acres to be seeded value and are
 			// not deleted
 			// It's possible that commodities are not specified
+			// Only commodities that are crop insurance eligible AND inventory crops are stored indiviually
+			// all other commodities are saved as OTHER
 			if (inventoryContractCommodity.getCropCommodityId() == null) {
 				unseededAcres = field.getPlantings().stream()
-						.filter(x -> x.getInventoryUnseeded().getCropCommodityId() == null
+						.filter(x -> (x.getInventoryUnseeded().getCropCommodityId() == null
+										|| x.getInventoryUnseeded().getCropCommodityId() != null
+										&& (Boolean.FALSE.equals(x.getInventoryUnseeded().getIsCropInsuranceEligibleInd())
+											|| Boolean.FALSE.equals(x.getInventoryUnseeded().getIsInventoryCropInd())))
```

Comment:
Maybe I don't understand... but on Line 507 and 508 you have 
`x.getInventoryUnseeded().getCropCommodityId() == null
|| x.getInventoryUnseeded().getCropCommodityId() != null`
Or there should be additional brackets?

Reply:
Good catch, I fixed it.